src/pytest_regressions/dataframe_regression.py

```diff
@@ -183,12 +185,12 @@ def _dump_fn(self, data_object, filename):
 
     def check(
         self,
-        data_frame,
-        basename=None,
-        fullpath=None,
-        tolerances=None,
-        default_tolerance=None,
-    ):
+        data_frame: Any,
+        basename: Optional[str] = None,
+        fullpath: Optional["os.PathLike[str]"] = None,
+        tolerances: Optional[Dict[str, Dict[str, float]]] = None,
```

Comment:
The inner dict here could have been an attrs class, this is something we could improve in the future. The old documentation even hinted at an unexistent `Tolerance` class...

Reply:
Yeah thought about it too, but didn't want to change the API at the same time.